ui/tsconfig.json

```
@@ -1,13 +1,14 @@
 {
   "include": ["**/*", "**/.server/**/*", "**/.client/**/*", ".react-router/types/**/*"],
```

Question (since I'm unfamiliar with Vite): can the `.react-router/types/**/*` be removed now as well?
```suggestion
  "include": ["**/*", "**/.server/**/*", "**/.client/**/*"],
```

Not sure if that was a remnant of the react-router test setup, or something from the default Vite config. Calling it out just in case 🙏

---

Yes this was a remnant, I wasn't 100% sure if the .react-router folder was still under the hood. I don't believe there's a use for it any more, `yarn start/build/preview` all work as expected without it and we don't explicitly use it.